src/error.rs

```diff
@@ -699,6 +712,33 @@ impl std::fmt::Display for CertError {
 
 impl error::Error for CertError {}
 
+#[derive(Debug)]
+/// Errors which may be encountered when handling attestation reports
+pub enum AttestationError {
```

Comment:
```suggestion
pub enum AttestationReportError {
```

Perhaps this is a bit more descriptive?